atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -201,7 +203,13 @@ def get_cluster_info(self, cluster, platform):
 
         conf = Configuration(**kwargs)
         osbs = OSBS(conf, conf)
-        current_builds = self.get_current_builds(osbs)
+        try:
+            current_builds = self.get_current_builds(osbs)
+        except OsbsException:
+            self.log.exception("Error occurred while listing builds on %s" %
```

Comment:
Preferred way is to use: `self.log.exception("...%s...", cluster.name)`
But it's really minor.